node/node.go

```diff
@@ -119,69 +125,6 @@ func NewNode(ctx context.Context, conf config.NodeConfig, nodeKey crypto.PrivKey
 	return node, nil
 }
 
-func (n *Node) mempoolReadLoop(ctx context.Context) {
-	for {
-		select {
-		case tx := <-n.incomingTxCh:
```

Comment:
Is the `incomingTxCh` unused now?

Reply:
that is a great point, currently yes it is. 

@tzdybal do you think we can remove that, or do we have other plans for it? What about `GossipHandler`s as well, do we still need those?

Perhaps we can address those in the future proposals that address #100 ?

Reply:
My comment was discarded somehow O_o

I think it's reasonable, to keep handler and validator separate (see https://github.com/celestiaorg/optimint/issues/100#issuecomment-921679366).

Reply:
regarding the initial question: `incomingTxCh`is still unused now. @tzdybal are you advocating to keep it because keeping the "handler and validator separate" would reintroduce that channel?